docs/sample_config.yaml

```diff
@@ -1186,7 +1186,7 @@ oembed:
 # connect to arbitrary endpoints without having first signed up for a
 # valid account (e.g. by passing a CAPTCHA).
 #
-#turn_allow_guests: true
+#turn_guests: true
```

Comment:
What's going on here? This sounds like it's related to WebRTC media rather than saml.

Reply:
I don't think I meant to touch that >.<
Sorry for being such a mess to work with..

Reply:
No worries, it's easily done!